src/base/17.0.1.3/attr_domains2expr.py

```diff
@@ -58,14 +58,14 @@ def adapt_view(cr, view_xmlid):
     IrUiView = util.env(cr)["ir.ui.view"].with_context(lang=lang)
     ```
     """
-    vid = util.ref(view_xmlid)
+    vid = util.ref(cr, view_xmlid)
     IrUiView = util.env(cr)["ir.ui.view"]
     view = IrUiView.browse(vid)
 
     # disable view to avoid it being applied to the parent arch
     view.active = False
     # get combined arch of the parent view
-    comb_arch = view.inherit_id._get_combined_arch()
+    comb_arch = view.inherit_id._get_combined_arch() if view.inherit_id else False
```

Comment:
Actually from the usage -- see [line 145](https://github.com/odoo/upgrade-util/blob/de2b64766edcf38294a840f143ebaabe0a446014/src/base/17.0.1.3/attr_domains2expr.py#L145) -- we should use `None` instead of `False` here.
```suggestion
    comb_arch = view.inherit_id._get_combined_arch() if view.inherit_id else None
```

Reply:
Oki dukey Done